cmd/flags/driver.go

```diff
@@ -29,11 +31,11 @@ var (
 		Value:    false,
 		Category: driverCategory,
 	}
-	P2PSyncTimeout = &cli.UintFlag{
+	P2PSyncTimeout = &cli.DurationFlag{
 		Name: "p2p.syncTimeout",
 		Usage: "P2P syncing timeout in seconds, if no sync progress is made within this time span, " +
```

Comment:
think we can remove "in seconds" in here and replace with an example format, ie "1h" now